packages/hop-node/src/utils/getRedundantRpcUrls.ts

```diff
@@ -0,0 +1,7 @@
+import { config as globalConfig } from 'src/config'
+
+const getRedundantRpcUrls = (network: string): string[] | undefined => {
+  return globalConfig.networks[network]?.redundantRpcUrls
```

Comment:
suggestion: return empty array by default

```suggestion
  return globalConfig.networks[network]?.redundantRpcUrls ?? []
```